src/nvim/marktree_defs.h

```diff
@@ -68,6 +68,7 @@ typedef struct {
   uint32_t id;
   uint16_t flags;
   DecorInlineData decor_data;  // "ext" tag in flags
+  bool scoped;
```

Comment:
Is tabpage or other scopes a possible need in the future? If so, should this be a string `'win'|'tab'|...` or int instead of bool?

Reply:
This is a member of a struct, so it definitely be as small as possible.

Reply:
I think a boolean is enough that as well.